### ISSUE TITLE
Possible error in the significance calculation

### ISSUE THREAD

**Issue by lucabaldini** [lucabaldini/ixpeobssim#709]
Hi Luca,
While doing the analysis for SS 433, several people were wondering why the significance from xpbinview disagrees with an independent calculation.  That significance comes from the (well named) routine significance.  I think that the line

``` 
sig[mask] =  scipy.stats.norm.ppf(conf[mask])
```

needs to be changed.  The norm function in scipy.stats extends to negative as well as positive values. 

For example, scipy.stats.norm.ppf(0.5), returns  0.0, since the integral from -infinity to zero equals that from zero to +infinity.

One should take this into account, so the line should be

```
sig[mask] =  scipy.stats.norm.ppf(conf[mask]/2+0.5)
```

I wrote a little code to test the original and suggested code for a few examples.
Thanks, Phil

 

 
```
# test significance calculation

import scipy

 

# common examples from https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule
# uncomment one (note, they are not quite as accurate as python doubles)
# p = 0.682689492137086 # 1 sigma
# p = 0.954499736103642 # 2 sigma
p = 0.997300203936740 # 3 sigma
# p = 0.999936657516334 # 4 sigma

# as implemented in significance routine in ixpeobssim
signow = scipy.stats.norm.ppf(p)
# how should be implemented since norm extends to negative as well as positive values
sigout = scipy.stats.norm.ppf(p/2+0.5)
print(sig, p, signow, sigout)
```

**Comment by lucabaldini** [comment]
Getting started on this one.

**Comment by lucabaldini** [comment]
Pull request: https://github.com/lucabaldini/ixpeobssim/pull/718

**Comment by lucabaldini** [comment]
Shipped in 30.6.3
